libs/designsystem/src/lib/components/tabs/tabs.component.spec.ts

```diff
@@ -96,4 +96,49 @@ describe('TabsComponent', () => {
       });
     });
   });
+
+  describe('Tab-bar height', () => {
```

Comment:
As it's almost identical tests, consider simplifying by creating a map of screensizes and expected height, and loop over that map instead:
```suggestion
  describe('Tab-bar height', () => {
    const scenarios: { [key in ScreenSize]?: string } = {
      phone: '50px',
      tablet: '70px',
      desktop: '70px',
    };

    Object.entries(scenarios).forEach(([screenSize, expectedHeight]) => {
      it(`should have correct height on screensize ${screenSize}`, async () => {
        await TestHelper.resizeTestWindow(TestHelper.screensize[screenSize]);
        
        const ionTabBarElm = spectator.query('ion-tab-bar');
        expect(ionTabBarElm).toHaveComputedStyle({
          height: expectedHeight,
        });
      });
    });
```

Reply:
Thanks. I like the suggestion. I'll update the PR.